aws/resource_aws_acm_certificate_validation.go

```diff
@@ -80,6 +80,16 @@ func resourceAwsAcmCertificateValidationCreate(d *schema.ResourceData, meta inte
 		log.Printf("[INFO] ACM Certificate validation for %s done, certificate was issued", certificate_arn)
 		return resource.NonRetryableError(resourceAwsAcmCertificateValidationRead(d, meta))
 	})
+	if isResourceTimeoutError(err) {
+		resp, err = acmconn.DescribeCertificate(params)
+		if *resp.Certificate.Status != "ISSUED" {
```

Comment:
To prevent potential panics we should prefer to use the [AWS Go SDK provided conversion functions](https://docs.aws.amazon.com/sdk-for-go/api/aws/) and the [available ACM service constants](https://docs.aws.amazon.com/sdk-for-go/api/service/acm/#pkg-constants) for consistency, e.g.

```suggestion
		if aws.StringValue(resp.Certificate.Status) != acm.CertificateStatusIssued {
```